README.rst

```diff
@@ -38,11 +38,11 @@ Current development version (development branch): |Docs Status dev| |Travis dev
 .. |Docs Status dev| image:: https://readthedocs.org/projects/icepyx/badge/?version=development
    :target: https://icepyx.readthedocs.io/en/development
 
-.. |Travis main Build Status| image:: https://app.travis-ci.com/icesat2py/icepyx.svg?branch=main
-    :target: https://app.travis-ci.com/icesat2py/icepyx
+.. |Tests Status (main)| image:: https://github.com/icesat2py/icepyx/actions/workflows/integration_test.yml/badge.svg?branch=main
+    :target: https://github.com/icesat2py/icepyx/actions/workflows/integration_test.yml
 
-.. |Travis dev Build Status| image:: https://app.travis-ci.com/icesat2py/icepyx.svg?branch=development
-    :target: https://app.travis-ci.com/icesat2py/icepyx
+.. |Tests Status (dev)| image:: https://github.com/icesat2py/icepyx/actions/workflows/integration_test.yml/badge.svg?branch=development
+    :target: https://github.com/icesat2py/icepyx/actions/workflows/integration_test.yml
```

Comment:
I don't suppose there's a magical way for this to check both action statuses (i.e. we don't want it to show as passing if the non-integration tests fail)?

Reply:
There is! https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/monitoring-workflows/adding-a-workflow-status-badge#using-the-event-parameter

Reply:
There isn't a way that I know of. The document you linked is just for filtering by event type. So we could show a badge for the status of the last integration test caused by a push event to the main branch.

If we want to show a combined badge for integration tests and unit tests, we will need to combine the workflows, to the best of my knowledge.

Reply:
What about a separate badge for each class of tests?

Reply:
I read the docs too literally, leading me to my not-quite-right interpretation!

> If we want to show a combined badge for integration tests and unit tests, we will need to combine the workflows, to the best of my knowledge. With the complexity of our triggers, and the future need to approve integration tests from forks, I don't want to do that 😆

100%. We should not combine the test workflows for badging purposes!

> What about a separate badge for each class of tests?

I think that's probably the way to go.

Reply:
Lines 10 and 12:
```
Latest release (main branch): |Docs Status main| |Unit Tests Status (main)| |Integration Tests Status (main)| |Code Coverage main|
Current development version (development branch): |Docs Status dev| |Unit Tests Status (dev)| |Integration Tests Status (dev)| |Code Coverage dev| |Pre-commit dev|
```

Reply:
I think we've got this now, please unresolve if you disagree :)